src/applications/vaos/containers/DateTimeSelectPage.jsx

```diff
@@ -103,6 +108,13 @@ export class DateTimeSelectPage extends React.Component {
     return (
       <div>
         {title}
+        <WaitTimeAlert
+          preferredDate={preferredDate}
+          nextAvailableApptDate="2020-01-20"
```

Comment:
Is this a temporary date?

Reply:
Forgot to remove a hardcoded date I used for testing.  🤦🏻‍♂️ Good catch.